lib/hoststats/collector.go

```diff
@@ -109,11 +109,13 @@ func (c *Collector) collect() {
 	hs.CPU = cpus
 
```

Comment:
For reference, we check if the `DataDirStats` is `nil` before using, so I didn't add a default value.

https://github.com/hashicorp/consul/blob/57bad0df85ddaef0d9120bc65b3d50db12bee746/lib/hoststats/host.go#L52